main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
-def main() -> QuerySet:
-    pass
+def main() -> None:
+    list_genre = ["Western", "Action", "Dramma"]
+    dict_actors = [
+        {"first_name": "George", "last_name": "Clooney"},
+        {"first_name": "Keanu", "last_name": "Reeves"},
+        {"first_name": "Scarlett", "last_name": "Keegan"},
+        {"first_name": "Will", "last_name": "Smith"},
+        {"first_name": "Jaden", "last_name": "Smith"},
+        {"first_name": "Scarlett", "last_name": "Johansson"}
+    ]
+
+    # Create GENRE with list genre
+
+    for genre in list_genre:
+        Genre.objects.create(genre=genre)
+
+    # Create ACTORS with dict actors
+
+    for actor in dict_actors:
+        Actor.objects.create(**actor)
+
+    # UPDATE genre Dramma, set name to "Drama"
+    # actor George Klooney, set last_name to "Clooney"
+    # actor Kianu Reaves, set first_name to "Keanu" and last_name to "Reeves"
+
+    Genre.objects.filter(genre="Dramma").update(genre="Drama")
```

Comment:
The `Genre.objects.filter()` and `update()` methods are using `genre="Dramma"` and `genre="Drama"`, which is incorrect. The correct field name is `name`, so it should be `name="Dramma"` and `name="Drama"`.